system/include/SSE/nmmintrin.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+#ifndef __emscripten_nmmintrin_h__
+#define __emscripten_nmmintrin_h__
+
+#ifndef __SSE4_2__
+#error "SSE4.2 instruction set not enabled"
+#endif
+
+#include <smmintrin.h>
+
+static __inline__ __m128i __attribute__((__always_inline__, __nodebug__))
+_mm_cmpgt_epi64(__m128i __a, __m128i __b)
+{
+  return (__m128i)((__i64x2)__a > (__i64x2)__b);
```

Comment:
Can we use the intrinsic for this comparison?

Reply:
There does not exist an intrinsic for i64x2 comparison in wasm_simd128.h header?

Reply:
Oh my bad, I forgot that we don't have i64x2 comparisons.